docs/content/latest/api/ysql/commands/cmd_copy.md

```diff
@@ -63,6 +70,41 @@ Specify the path of the file to be copied. An input file name can be an absolute
 
 ## Examples
 
+Assuming we have a table like below:
+
+```postgresql
+yugabyte=# CREATE TABLE users(id BIGSERIAL PRIMARY KEY, name TEXT);
+yugabyte=# INSERT INTO users(name) VALUES ('John Doe'), ('Jane Doe'), ('Dorian Gray');
+yugabyte=# SELECT * FROM users;
+ id |    name     
+----+-------------
+  3 | Dorian Gray
+  2 | Jane Doe
+  1 | John Doe
+(3 rows)
+```
+
+### Export the whole table
+Copy the whole table to a comma separated CSV file with column names in the header.
+
+```postgresql
+yugabyte=# COPY users TO '/home/yuga/Desktop/users.txt.sql' DELIMITER ',' CSV HEADER;
+```
+
+### Export partial table using WHERE clause with column selection
+Use a WHERE clause to filter rows and only `name` column.
+
+```postgresql
+yugabyte=# COPY (SELECT name FROM users where name='Dorian Gray') TO '/home/yuga/Desktop/users.txt.sql' DELIMITER
+ ',' CSV HEADER;
+```
+
+### Importing the previous exports
```

Comment:
```suggestion
### Import from CSV files

In the following example, the data exported in the previous examples are imported.

```

Reply:
Mention where the data is imported. If you're copying the data to a different table, mention it.

Reply:
mentioned it